lib/CFRPageActions.jsm

```diff
@@ -8,12 +8,23 @@ const { XPCOMUtils } = ChromeUtils.import(
 );
 const { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
 XPCOMUtils.defineLazyGlobalGetters(this, ["fetch"]);
+ChromeUtils.defineModuleGetter(this, "OS", "resource://gre/modules/osfile.jsm");
```

Comment:
nit: could you please group all `moduleGetter` calls under 1 `XPCOMUtils.defineLazyModuleGetters` call instead?